### ISSUE TITLE
V1.0.0 relying on deprecated React features.

### ISSUE THREAD

**Issue by SevenZark** [zenoamaro/react-quill#240]
I just installed 1.0.0 and it appears to be using deprecated features of React. That should probably be addressed at some point.

I've attached the warnings I see in my console via a captured image since I can't copy/paste the text, unfortunately.

![image](https://user-images.githubusercontent.com/426718/28679943-609ace52-72ba-11e7-80ad-489174aedb55.png)

**Comment by SevenZark** [comment]
Whoops, accidentally saved before entering my issue. So this is it. I just installed 1.0.0 and it appears to be using deprecated features of React. That should probably be addressed at some point.

I've attached the warnings I see in my console via a captured image since I can't copy/paste the text, unfortunately.

![image](https://user-images.githubusercontent.com/426718/28679885-33356972-72ba-11e7-87f9-ba89b9b2efea.png)
